src/modules/pmc/src/lib/Pmc.cpp

```diff
@@ -2,8 +2,35 @@
 // Licensed under the MIT License.
 
 #include "Pmc.h"
+#include <CommonUtils.h>
+#include <Mmi.h>
+#include <string>
 
 Pmc::Pmc(unsigned int maxPayloadSizeBytes)
     : PmcBase(maxPayloadSizeBytes)
 {
+}
+
+int Pmc::RunCommand(const char* command, bool replaceEol, std::string* textResult, unsigned int timeoutSeconds)
+{
+    char* buffer = nullptr;
+    int status = ExecuteCommand(nullptr, command, replaceEol, true, 0, timeoutSeconds, &buffer, nullptr, PmcLog::Get());
+
+    if (status == MMI_OK)
+    {
+        if (buffer && textResult)
+        {
+            *textResult = buffer;
+        }
+    }
+    else if (IsFullLoggingEnabled())
+    {
+        OsConfigLogError(PmcLog::Get(), "RunCommand failed with status: %d and output '%s'", status, buffer);
+    }
+
```

Comment:
Why duplicate the log without adding any new information? commonutils logs. See if the module can add anything else useful.